tests/script-based-pre/stubbing_compiler_sessions/stubbing.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Check that Kani handle different sets of stubbing correctly.
```

Comment:
This description is the one I refer in the `expected` file. If the standard for Kani is to leave in the test header, it's fine. We mean "handle correct" by "compiling all proof harness with same set of stubs in a single run". Is there a way we can check that they run in a single run using the `expected` file?

Reply:
By correctly I mean that the result is correct, i.e., that all harnesses behave as expected given the set of stubs that are specified on them.